### PR TITLE
Move sources to use generated client

### PR DESCRIPTION
**Pull request by bthesorceror**
- [ ] Rebased/mergeable
  - [ ] Tests pass
  - [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/sources/actions/index.ts [hunk 1]
```diff
@@ -91,7 +88,7 @@ export const createSource = (attrs: Partial<Source>) => async (
 export const updateSource = (source: Source) => async (
   dispatch: Dispatch<Action>
 ) => {
-  const updatedSource = await updateSourceAJAX(source)
+  const updatedSource = await updateSourceAJAX(source.id, source)
```
Comment (ebb-tide) [inline ui/src/sources/actions/index.ts]: hrrm. Isn't it weird that this function takes both `source` and `source.id`?

Comment (bthesorceror) [reply ui/src/sources/actions/index.ts]: I could see it either way but the underlying client does require an id and an updated source.

File: ui/src/sources/apis/index.ts [hunk 2]
```diff
@@ -1,57 +1,40 @@
...
+  const {data} = await sourcesAPI.sourcesGet('')
 
   return data.sources
 }
 
-export const readSource = async (url: string): Promise<Source> => {
-  const {data: source} = await AJAX({
-    url,
-  })
+export const readSource = async (id: string): Promise<Source> => {
```
[the remark this answers is not in the document]
Comment (bthesorceror) [reply ui/src/sources/apis/index.ts]: 👍

[the remark this answers is not in the document]
Comment (bthesorceror) [reply ui/src/sources/apis/index.ts]: Since this is a non minor change I will leave it to another PR

**Pull request merged**